openpgp/packet/reader.go

```diff
@@ -42,6 +43,7 @@ func (r *Reader) Next() (p Packet, err error) {
 			r.readers = r.readers[:len(r.readers)-1]
 			continue
 		}
+		// TODO: Refuse unknown packets, instead of ignoring them.
```

Comment:
This is very nitpicky but I would say something like
```suggestion
		// TODO: Add strict mode that rejects unknown packets, instead of ignoring them.
```
to make it clear that we probably can't do this by default.